### PR TITLE
Creating JSON Schema representations of DRO, Collection, File, Fileset MAPs

### PR DESCRIPTION
**Pull request by cmh2166**
This is a first pass of what will be living specifications. It will be and is meant to be added to. 

Please review this in tandem with https://github.com/sul-dlss-labs/taco/wiki/TACO-Data-Modeling-&-MAPs and see if this is okay for a starting point.

### CODE REVIEW

File: maps/DRO.json [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+            "description": "",
+            "type": "string"
+          },
+          "preserve": {
+            "description": "Whether or not the DRO should be preserved.",
+            "type": "boolean",
+          },
+          "remediatedBy": {
+            "description": "The Agent (User, Group, Application, Department, other) that remediated a DRO in SDR.",
+            "type": "object",
```
Comment (jcoyne) [inline maps/DRO.json]: Could this be an array? Could it have undergone multiple remediations?

Comment (cmh2166) [reply maps/DRO.json]: WIP.

Comment (jcoyne) [reply maps/DRO.json]: Yes, I saw WIP. Can we collaborate on this? How/when would you like comments and questions?

File: docs/DRO.json [hunk 2]
```diff
@@ -2,149 +2,250 @@
...
         "type": "string"
       },
-      "catkey": {
-        "description": "Catalog bibliographic record identifier (or ‘catkey’) for the same digital object being described by the bibliographic record.",
-        "type": "string"
+      "depositor": {
+        "description": "The Agent (User, Group, Application, Department, other) that deposited the DRO into SDR.",
+        "type": "object",
+        "properties": {
+          "required": ["name"],
```
[the remark this answers is not in the document]
Comment (cmh2166) [reply docs/DRO.json]: not relevant. comment made before completion of first pass.

File: maps/DRO.json (hunk 1 above)
Comment (cmh2166) [reply maps/DRO.json]: no longer relevant. comment made before completion of first pass.

**Comment by jcoyne** [comment]
@cmh2166 did you mean to wait for *all* of the requested reviews or should we merge now we have 3?

**Pull request merged**